src/game/client/components/menus.cpp

```diff
@@ -2502,7 +2502,8 @@ void CMenus::RenderBackground()
 	Graphics()->TextureClear();
 	Graphics()->QuadsBegin();
 		float Size = 15.0f;
-		float OffsetTime = fmod(Client()->LocalTime()*0.15f, 2.0f);
+		static int64 s_MenuBackgroundStart = time_get();
+		float OffsetTime = fmod((time_get()-s_MenuBackgroundStart)*0.15f/time_freq(), 2.0f);
```

Comment:
Try not to use time_get in a render function

Reply:
What else to use thou? Or where to move this timer? `Client()->LocalTime()` is not running at that point because `RenderBackground` is called by `RenderLoading` which is running outside of the game/render loop that updates the local time in [client.cpp#L2141](https://github.com/Robyt3/teeworlds/blob/afda8d5923c60fa381de9c5f912579d74b5cad17/src/engine/client/client.cpp#L2141). `IClient` would need to expose a "UpdateLocalTime" method to the gameclient so it can update the local time itself from `RenderLoading`.

Reply:
Hm, the whole thing is kinda hacky as you hijack the rendering process in this pr. And f.e. don't use the double buffering which might lead to problems for some users.
About the timer: You could transfer it to the RenderLoading function which is only used during the initilaization, and so keep the normal process clean. That includes the [Clear ](https://github.com/teeworlds/teeworlds/pull/2459/files#diff-cad6690c312d4ab80e91a046cdbb5becR2495) call too, which shouldn't be there during the normal process.